cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,15 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+  const EMAIL = 'gemowerez@mailinator.com';
```

Comment:
```suggestion
  const email = 'gemowerez@mailinator.com';
```
use camelCase to declare the variables